pyramid-app/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/db.py

```diff
@@ -0,0 +1,40 @@
+import sqlalchemy
+import zope.sqlalchemy
+from sqlalchemy import MetaData
+from sqlalchemy.orm import DeclarativeBase, MappedAsDataclass, Session
+
+
+class Base(MappedAsDataclass, DeclarativeBase):
+    metadata = MetaData(
+        naming_convention={
+            "ix": "ix_%(column_0_N_label)s",
+            "uq": "uq_%(table_name)s_%(column_0_N_name)s",
+            "ck": "ck_%(table_name)s_%(constraint_name)s",
+            "fk": "fk_%(table_name)s_%(column_0_N_name)s_%(referred_table_name)s_%(referred_column_0_N_name)s",
+            "pk": "pk_%(table_name)s",
+        }
+    )
+
+
+def create_engine(settings):  # pragma: no cover
+    """Return a SQLAlchemy engine."""
+    return sqlalchemy.create_engine(settings["database_url"])
```

Comment:
This is extracted into a separate function because it's called by `includeme()` below and is also imported and called by `bin/make_db`